api/signup.go

```diff
@@ -7,12 +7,14 @@ type SignupEnabledResponse struct {
 }
 
 type SignupRequest struct {
+	OrgName  string `json:"orgName"`
```

Comment:
I wasn't sure if we should pollute the existing signup process or create a new one. I think we should also be asking for the domain?